Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.yml

```diff
@@ -0,0 +1,1179 @@
+commonfields:
+  id: VirusTotal v3
+  version: -1
+name: VirusTotal v3
+display: VirusTotal v3
+fromversion: 5.5.0
+category: Data Enrichment & Threat Intelligence
+description: Analyze suspicious hashes, URLs, domains and IP addresses
+configuration:
```

Comment:
@jochman Please remember to add the reliability param as described here https://github.com/demisto/etc/issues/29347
and use it in the DBotScore classes(DBotScoreReliability).
you can approach @JasBeilin for further info.